xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml

```diff
@@ -1100,7 +1100,7 @@ For more information about PBKDF2, see [RFC 2898](https://www.rfc-editor.org/inf
         <param name="salt">The key salt used to derive the key.</param>
         <param name="iterations">The number of iterations for the operation.</param>
         <param name="hashAlgorithm">The hash algorithm to use to derive the key.</param>
-        <param name="outputLength">The size of key to derive.</param>
+        <param name="outputLength">The size of key to derive (that is, the number of elements in the byte array).</param>
```

Comment:
I find the parenthetical very odd.

* The size of the key, in bytes, to derive.
* The size of the key to derive, in bytes.
* The length of the byte array to create and fill with the derived key.

Something to that effect.